modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java

```diff
@@ -209,17 +203,23 @@ public ServiceTemplateEntity registerServiceTemplateByUrl(String oclLocation) th
     /**
      * Get detail of service template using ID.
      *
-     * @param id the ID of
+     * @param id             the ID of
+     * @param operationType  the operation type. values:view,update,unregistered.
+     * @param checkNamespace check the namespace of the service template belonging to.
      * @return Returns service template DB newTemplate.
      */
-    public ServiceTemplateEntity getServiceTemplateDetails(String id, boolean checkNamespace) {
+    public ServiceTemplateEntity getServiceTemplateDetails(String id,
+                                                           String operationType,
```

Comment:
i'm not sure if we really need this. 
if the purpose is to just get a better log message.. we can just havea generic message saying "No permisions to view or manage service templates ...................................."

Reply:
Updated as suggested.